src/oss/confluent.sh

```diff
@@ -1433,6 +1479,65 @@ version_command() {
     fi
 }
 
+produce_usage() {
+    local exit_status="${1}"
+    if [[ -z "${exit_status}" ]]; then
+      exit_status=0
+    fi
+
+    cat <<EOF
+Usage: ${command_name} <topicname> [--value-format avro] [other optional args]
+
+Description:
+    Produce to Kafka topic specified by <topicname>.
+
+    Without '--value-format avro', it calls the 'kafka-console-producer' command,
+    automatically with '--topic <topicname> --broker-list localhost:9092' and passes in <other optional args>.
+
+    With '--value-format avro', it calls the 'kafka-avro-console-producer' command,
+    automatically with '--topic <topicname> --broker-list localhost:9092' and passes in <other optional args>.
+    This requires an additional argument '--property value.schema=<schema>'
+
+    To see the other optional args, call the underlying command line with '--help'
+
+Examples:
+    confluent produce mytopic1
+
+    confluent produce mytopic1 --value-format avro --property value.schema='{"type":"record","name":"myrecord","fields":[{"name":"f1","type":"string"}]}'
+
+EOF
+    exit $exit_status
+}
+
+consume_usage() {
+    local exit_status="${1}"
+    if [[ -z "${exit_status}" ]]; then
+      exit_status=0
+    fi
+
+    cat <<EOF
+Usage: ${command_name} <topicname> ] [--value-format avro] [other optional args]
+
+Description:
+    Consume from Kafka topic specified by <topicname>.
+
+    Without '--value-format avro', it calls the 'kafka-console-consumer' command,
+    automatically with '--topic <topicname> --bootstrap-server localhost:9092' and passes in <other optional args>.
```

Comment:
mention that `localhost` with kafka_port from the configuration is used and don't accept `--bootstrap-server`.

Reply:
It was intentional to provide `--bootstrap-server`, so it can be overridden to allow users to use the CLI to connect to a Kafka cluster that is not on localhost